server/index.js

```diff
@@ -18,63 +19,93 @@ app.use('*.js', (req, res, next) => {
 app.use(express.static('./client/dist'));
 app.use(express.json());
 
-/* ******** Personnel Routes ***************** */
+/* ******** Get Routes ***************** */
+
+app.get('/api/movies', async (req, res) => {
+  console.log('Received GET at /api/movies');
+  // find one movie object,
+  // find related personnel,
+  // populate movie object with personnel list.
+  let movie;
+  await db.getMovieById(Number(req.query.id))
```

Comment:
There's no need to be running 2 async operations in parallel when the first does not need to be completed before the second.

This is an excellent example of where `async/await` is just getting in the way, and where we might be better served by `Promise.all(...)` instead.